### ISSUE TITLE
ChipGroup - accessibility refinements

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#1454]
The following accessibility issues are present in the ChipGroup component:

- **Issue:** In the HTML that gets rendered, an `aria-label="Close"` attribute is defined on the "# More" and "Show Less" buttons. This label doesn't start with the text string that's visible, and it doesn't describe the action that's happening. In this case, since the button has visible text, `aria-label` shouldn't be used.
  - **Update:** remove `aria-label` attribute from the "# More" and "Show Less" buttons
- **Issue:** When the chip groups are expanded, there should only be one `<ul class=“pf-c-chip-group pf-m-toolbar”>` element that wraps all the first level list items (`<li><h*><ul></li>`), so that the number of chip groups are communicated to screen reader users. Currently, each chip group is a separate list, with one list item. 
  - **Update:**
    - Replace the following structure:
      ```
      <ul class="pf-c-chip-group pf-m-toolbar"
        <li>
          <h4> class="pf-c-chip-group__label">Category 1</h4>
          <ul class="pf-c-chip-group">...</ul>
        </li>
      </ul>
      <ul class="pf-c-chip-group pf-m-toolbar"
        <li>...</li>
      </ul>
      <ul class="pf-c-chip-group pf-m-toolbar"
        <li>...</li>
      </ul>
      ```
    - With the following structure:
      ```
      <ul class="pf-c-chip-group pf-m-toolbar"
        <li>
          <h4> class="pf-c-chip-group__label">Category 1</h4>
          <ul class="pf-c-chip-group">...</ul>
        </li>
        <li>...</li>
        <li>...</li>
      </ul>
      ```
- **Issue:** Heading level is set to `<h4>`, but heading level should be defined by the consumer relative to context in which this component is used. 
  - **Update:** Allow the consumer to set the heading level in the chip group.

**Comment by jgiardino** [comment]
Closing this issue. Some updates requested are no longer relevant due to updates to core (e.g. specifying a heading level), and other updates have been addressed.
